### ISSUE TITLE
The code for creating the dataset.

### ISSUE THREAD

**Issue by Avon-C** [FangHanNUS/MM22_PIMoG#9]
Hello, author. I noticed a significant difference between the edge masks produced by the BDCN mentioned in the paper and the examples you provided. Could you please share the code for generating the dataset? I am interested in training your model for comparative experiments. Thank you very much!

**Comment by YinDFY** [comment]
Hi, have you got a pretrained BDCN and code to generate the dataset? If you have already addressed this problem, can you share the code and model with me?

**Comment by zy-abdl** [comment]
> Hi, have you got a pretrained BDCN and code to generate the dataset? If you have already addressed this problem, can you share the code and model with me?

Has your problem been solved and if so, can you please share your code?
